### ISSUE TITLE
[pact-jvm-consumer-groovy] MissingMethodException when running a pact test

### ISSUE THREAD

**Issue by artamonovkirill** [DiUS/pact-jvm#863]
Since version 3.6.2, Groovy consumer Pact tests are failing with:
```
Assertion failed: 

assert result == PactVerificationResult.Ok.INSTANCE  // This means it is all good
       |      |                            |
       |      false                        au.com.dius.pact.consumer.PactVerificationResult$Ok@14ffc5c7
       Error(error=groovy.lang.MissingMethodException: No signature of method: com.github.artamonovkirill.AliceServiceConsumerPactTest$_A_service_consumer_side_of_a_pact_goes_a_little_something_like_this_closure2.doCall() is applicable for argument types: (au.com.dius.pact.consumer.MockHttpServer, au.com.dius.pact.consumer.PactTestExecutionContext) values: [au.com.dius.pact.consumer.MockHttpServer@6c084e2c, PactTestExecutionContext(pactFolder=target/pacts)]
       Possible solutions: doCall(), doCall(java.lang.Object), findAll(), findAll(), mockServerState=ExpectedButNotReceived(expectedRequests=[	method: get
       	path: /mallory
       	query: [:]
       	headers: [:]
       	matchers: MatchingRules(rules={})
       	generators: Generators(categories={})
       	body: MISSING]))
```

Check an example [here](https://github.com/artamonovkirill/pact-jvm-consumer-groovy-test) (a slightly modified version of test from pact-jvm-consumer-groovy [README](https://github.com/DiUS/pact-jvm/tree/master/pact-jvm-consumer-groovy#usage))

**Comment by artamonovkirill** [comment]
It works if I add `service` and `context` to runTest closure: `runTest { server, context ->`.
Is it possible to make `runTest` backwards compatible with 3.6.1 and before?

**Comment by uglyog** [comment]
Done

**Comment by uglyog** [comment]
This fix has been released with version 3.6.4
